cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -0,0 +1,219 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/google/keytransparency/core/authentication"
+
+	"github.com/aybabtme/uniplot/histogram"
+	"github.com/paulbellamy/ratecounter"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	tm "github.com/buger/goterm"
+	tpb "github.com/google/keytransparency/core/api/type/type_proto"
+)
+
+var (
+	maxWorkers int
+	workers    int
+	memLog     = new(bytes.Buffer)
+	ramp       time.Duration
+)
+
+func init() {
+	log.SetOutput(memLog)
+
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+
+	RootCmd.AddCommand(hammerCmd)
+
+	hammerCmd.Flags().IntVar(&maxWorkers, "workers", 1, "Number of parallel workers")
+	hammerCmd.Flags().DurationVar(&ramp, "ramp", 1*time.Second, "Time to spend ramping up")
+}
+
+// hammerCmd represents the post command
+var hammerCmd = &cobra.Command{
+	Use:   "hammer",
+	Short: "Loadtest the server",
```

Comment:
What server?

Reply:
The server target and other authentication variables are defined in `root.go`

Reply:
It looked like this short message could be shown to the user and I was just questioning whether it was useful.

Reply:
It's not the most useful.  I'll add the following longer description: 
```
Sends update requests for user_1 through user_n using a select number of workers in parallel
```